changelog.txt

```diff
@@ -1,4 +1,7 @@
-0.83.1
+0.84.0
```

Comment:
Yes, but we need to rewrite the changelog before a release anyway. As this PR must be rebased on top of 0.83.0. For merging to master I think 0.84.0 is the correct version?